shared_libraries/orca_shared/database/use_cases/validation.py

```diff
@@ -10,7 +10,6 @@ def validate_config(config: PostgresConnectionInfo, logger: logging.Logger) -> N
     validate_postgres_name(config.admin_username, "Admin username", logger)
 
     _validate_password(config.user_password, "User", logger)
```

Comment:
Ideally we'd cover the db_admin password as well (so when we decouple we already have the checks in-place), but I haven't looked into the Cumulus password requirements, so I don't know if this is feasible.